resources/profiles/Elegoo/machine/ECC2/Elegoo Centauri Carbon 2 0.4 nozzle.json

```diff
@@ -0,0 +1,83 @@
+{
+  "type": "machine",
+  "name": "Elegoo Centauri Carbon 2 0.4 nozzle",
+    "inherits": "fdm_machine_ecc",
+  "from": "system",
+  "setting_id": "ECC204",
+  "instantiation": "true",
+  "nozzle_diameter": [
+    "0.4"
+  ],
+  "host_type": "elegoolink",
+  "printer_model": "Elegoo Centauri Carbon 2",
+  "printer_variant": "0.4",
+  "auxiliary_fan": "1",
+  "printable_area": [
+    "0x0",
+    "256x0",
+    "256x256",
+    "0x256"
+  ],
+  "printable_height": "256",
+  "retract_lift_below": [
+    "255"
+  ],
+  "bed_exclude_area": [
+    "246x0",
+    "256x0",
+    "256x20",
+    "246x20"
+  ],
+  "bed_texture_area": [
+    "0x-10",
+    "256x-10",
+    "256x256",
+    "0x256"
+  ],
+  "thumbnails": [
+    "144x144"
+  ],
+  "machine_max_acceleration_travel": [
+    "20000",
+    "20000"
+  ],
+  "default_filament_profile": [
+    "Elegoo PLA @ECC2"
+  ],
+  "default_print_profile": "0.20mm Standard @Elegoo CC2 0.4 nozzle",
+  "extruder_offset": [
+    "0x1.5"
+  ],
+  "default_bed_type": "4",
+  "fan_speedup_time": "0.5",
+  "machine_load_filament_time": "29",
+  "machine_unload_filament_time": "28",
+  "nozzle_type": "hardened_steel",
+  "scan_first_layer": "1",
+  "upward_compatible_machine": [],
+  "manual_filament_change": "0",
+  "auto_toolchange_command": "0",
+  "purge_in_prime_tower": "0",
+  "gcode_flavor": "klipper",
+  "machine_pause_gcode": "M600",
```

Comment:
FYI, I don't know how far the CC2 has veered from the CC but this probably shouldn't be an `M600` code and should be a `PAUSE`. I just opened up a PR to fix this for the CC.